src/main/java/org/fcrepo/spec/testsuite/authz/WebACRdfSources.java

```diff
@@ -50,8 +51,20 @@ public WebACRdfSources(final String username, final String password) {
     @Parameters({"param1"})
     public void aclIsLDPRS(final String uri) {
         final TestInfo info = setupTest("5.1", "aclIsLDPRS",
-                "An ACL for a controlled resource on a conforming server must itself be an LDP-RS.",
-                "https://fedora.info/2018/06/25/spec/#solid-ldp-acls", ps);
+                                        "An ACL for a controlled resource on a conforming server must itself be an " +
+                                        "LDP-RS.",
+                                        "https://fedora.info/2018/06/25/spec/#solid-ldp-acls", ps);
+
+        //attempt to create a LDP-NR at an acl location and confirm that it fails.
+
+        //create resource
+        final String resourceUri = createResource(uri, info.getId());
+        //get acl handle
+        final String aclUri = getAclLocation(resourceUri);
+        //create non-rdf acl
+        final Response response = doPutUnverified(aclUri, new Headers(new Header("Content-Type", "text/plain")),
```

Comment:
Additionally, I think this test should create an ACL and check the response headers on a subsequent `HEAD` that it is an LDP-RS.